.azure_pipelines/ci-eval-pr.yaml

```diff
@@ -51,4 +52,22 @@ jobs:
             echo "\n\nOutputting git diff for checked in files.\n"
             git diff
           fi
-        displayName: NoDiffCheck
\ No newline at end of file
+        displayName: NoDiffCheck
+      - bash: |
+          source activate $(condaEnvFileSuffix)
+
+          pip install pytest==7.0.1 pytest-azurepipelines 'langchain>=0.0.230' 'llama_index>=0.8.3'
```

Comment:
What's the plan for keeping these packages up to date?

Reply:
these are installed on fresh environment each time. so should be up to date.